lib/pay/services/service_gateway_account.ex

```diff
@@ -4,10 +4,12 @@ defmodule Pay.Services.ServiceGatewayAccount do
   @timestamps_opts [type: :utc_datetime_usec]
 
   schema "service_gateway_accounts" do
-    field :gateway_account_id, Ecto.UUID
-
     belongs_to :service, Pay.Services.Service
 
+    belongs_to :gateway_account, Pay.Payments.GatewayAccount,
```

Comment:
@dagoof are you going to revert this relationship and the query change above? If we are going to combine the domains then I'd prefer we did it in one go separately from here (and for example we'd probably just make the service/gateway account relationships use the integer ID not the external ID so there's more changes needed).

Reply:
will revert